src/constants/PATHS.ts

```diff
@@ -0,0 +1,6 @@
+export const PATHS = {
+  BASE: '/react-shopping-cart/',
```

Comment:
```suggestion
  BASE: '/react-shopping-cart',
```

뒤에도 `/`가 붙어있으면 path끼리 조합할 때 `//`가 생기는 경우가 있을 것 같아요!

Reply:
수정했습니다! 그런 경우를 생각을 못했네요 🥹